.ameba.yml

```diff
@@ -0,0 +1,194 @@
+  Excluded:
+  # Samples may legitimately use literals in conditions
+  - samples/**/*
+
+Lint/LiteralInInterpolation:
+  Excluded:
+  - spec/std/random_spec.cr # BUG: https://github.com/crystal-ameba/ameba/issues/611
+
+Lint/LiteralsComparison:
+  Excluded:
+  # Explicit tests for case equality on tuple literals
+  - spec/std/tuple_spec.cr
+
+# TODO: Investigate if some `not_nil!` calls can be avoided.
+Lint/NotNil:
+  Enabled: false
+
+Lint/RandZero:
+  Excluded:
+  # Explicit tests
+  - spec/std/random_spec.cr
+
+# FIXME: Resolve shadowing.
+Lint/ShadowingOuterLocalVar:
+  Enabled: false
+
+# TODO: Investigate file names and move to data directories.
+Lint/SpecFilename:
+  Enabled: false
+
+# BUG: https://github.com/crystal-ameba/ameba/issues/612
+Lint/TopLevelOperatorDefinition:
+  Enabled: false
+
+# Disabled: We have an explicit CI job for `typos`. No reason to run it through
+# ameba.
+Lint/Typos:
+  Enabled: false
+
+# TODO: Investigate unused arguments.
+Lint/UnusedArgument:
+  Enabled: false
+
+# TODO: Investigate unused block arguments.
+Lint/UnusedBlockArgument:
+  Enabled: false
+
+# FIXME: Investigate useless assigns.
+Lint/UselessAssign:
+  Enabled: false
+
+# Metrics
+# =========================
+
+# Disabled: Lot's of violations. Complexity is very individual.
+Metrics/CyclomaticComplexity:
+  Enabled: false
+
+# Naming
+# =========================
+# All disabled. There are many violations and some of the rules are questionable.
+# TODO: Consider enabling some of these rules.
+
+Naming/AccessorMethodName:
+  Enabled: false
+
+Naming/BinaryOperatorParameterName:
+  Enabled: false
+
+Naming/BlockParameterName:
+  Enabled: false
+
+# Disabled: All violations follow the spelling of identifiers in upstream
+# projects, e.g. for lib bindings.
+Naming/ConstantNames:
+  Enabled: false
+
+Naming/MethodNames:
+  Enabled: false
+
+Naming/PredicateName:
+  Enabled: false
+
+Naming/QueryBoolMethods:
+  Enabled: false
+
+Naming/RescuedExceptionsVariableName:
+  Enabled: false
+
+Naming/TypeNames:
+  Enabled: false
+
+Naming/VariableNames:
+  Enabled: false
+
+# Performance
+# =========================
+
+Performance/AnyInsteadOfEmpty:
+  Excluded:
+  # These specs explicitly test `#any?` implementations
+  - spec/std/bit_array_spec.cr
+  - spec/std/enumerable_spec.cr
+  - spec/std/hash_spec.cr
+
+# Style
+# =========================
+# All disabled. There are many violations and some of the rules are questionable.
+# TODO: Consider enabling some of these rules.
+
+Style/HeredocEscape:
+  Enabled: false
+
+Style/HeredocIndent:
+  Enabled: false
+
+Style/MultilineCurlyBlock:
+  Enabled: false
+
+# Disabled: This rule seems too strict when any negation inside a complex condition is
+# considered a violation. https://github.com/crystal-ameba/ameba/issues/621
+Style/NegatedConditionsInUnless:
+  Enabled: false
+
+# BUG: https://github.com/crystal-ameba/ameba/issues/614
+Style/ParenthesesAroundCondition:
+  Enabled: false
+
```

Comment:
ditto

```suggestion
```

Reply:
The fix is only in master. Latest ameba release is still broken.
There are also still other violations of this rule. This is a TODO for a follow-up.